app/controllers/api/v1/documents_controller.rb

```diff
@@ -34,7 +34,7 @@ def document_params
 
       def document_attributes
```

Comment:
as we are re-using this controller for 2 different (but very similar) endpoints, I think an intention comment would be useful.

Reply:
well, the intention is for the code to do exactly what it does... there's a ticket about this, the reasoning behind it is not entirely clear to me (and I don't think it's relevant)